openapi/workflow_execution_service.openapi.yaml

```diff
@@ -639,7 +644,33 @@ components:
           type: string
         state:
           $ref: '#/components/schemas/State'
-      description: Small description of a workflow run, returned by server during listing
+      description: State information of a workflow run
+    RunSummary:
+      title: RunSummary
+      allOf:
+        - $ref: '#/components/schemas/RunStatus'
+        - type: object
+          properties:
+            name:
```

Comment:
Similar to tags, I am largely in favour of this attribute, but I would probably want to ensure this is accessible by implementors before adding it

Reply:
Where would this come from? The current `RunRequest` doesn't have an explicit attribute for `name`. IIRC, the way we've handled it is with a specific `tag`. That said, if `name` is important enough to be used by other parts of the API, it might be worth elevating it to a top level property of `RunRequest`.

Reply:
It's a property of the [`Log` model](https://github.com/ga4gh/workflow-execution-service-schemas/blob/6e6327fa680cfb03d8e81bf594b7625b5348e011/openapi/workflow_execution_service.openapi.yaml#L731-L759):

```yaml
    Log:
      title: Log
      type: object
      properties:
        name:
          type: string
          description: The task or workflow name
```